### ISSUE TITLE
Screen Reader (narrator) announce wrong information(selection contain 1 items) for the  fabric DropDown even DropDown have no data

### ISSUE THREAD

**Issue by Sheepu-dev** [microsoft/fluentui#12012]
Screen reader announce wrong information for the dropdown control if Dropdown contains no data.

### Environment Information
- **Browser and OS versions**: Edge/Window10

### Codepen Path:

https://codepen.io/sheepu/pen/xxGELGq?editors=1010

#### Actual behavior:

SR announce that Dropdown contains one element even no data exists in Dropdown .

#### Expected behavior:

Screen Reader should announce "control contains zero element or no element" if no option or data available in Dropdown .

**Comment by aneeshack4** [comment]
Hi @Sheepu-dev, thanks for reaching out to us! I see that this repros even on the basic dropdown. I dug into the code a bit and investigated. There are a couple weird things: (1) it looks like `aria-setsize` is being properly set to 0: 
![image](https://user-images.githubusercontent.com/4161791/74970620-8868dd00-53d3-11ea-97c5-308e6677266e.png)
(2) I thought maybe the size cache used to keep track of the number of options in `Dropdown.base.tsx` might have a bug but it looks correct: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dropdown/utilities/DropdownSizePosCache.ts#L22. So hard to tell if this was always a bug or it started regressing at some point (leaning towards the first since this size cache code hasn't really changed in a while).

**Comment by Sheepu-dev** [comment]
Hi, @aneeshack4  thanks for updates.

**Comment by eivindland** [comment]
We also have some accessibility issues with DropDown. In a screen reader test it was reported that it behaved "strangely". Our accessiblity tester recommended to try changing role="listbox" to role="button" on div-element with med id="DropdownNNN" to make it behave more naturally.
